app/celery/tasks.py

```diff
@@ -105,6 +106,9 @@ def process_job(job_id, sender_id=None):
     job.processing_started = start
     dao_update_job(job)
 
+    # Record StatsD stats to compute SLOs
+    statsd_client.timing_with_dates('job.processing-start-delay', job.processing_started, job.scheduled_for)
```

Comment:
Slightly confused here, `timing_with_dates` as a prototype of (stat, start, end, rate=1) however logging it here isn't actually the end of the job.
Should we be logging this after the `job_complete` ?

Reply:
Our SLO is about the start of the job, not the end.

> Scheduled sends start on the time they were scheduled for.

Jobs can take a long time to complete if we're sending 40k notifications. What's important is that if people said they want those notifications to be delivered at 10am, we should start to send those at 10am

Reply:
ok go it, `processing-start-delay` - that makes sense